protos/api_upstream/.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # Syntax is here:
 # https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-*                @temporalio/server @temporalio/sdk
\ No newline at end of file
+* @temporalio/server @temporalio/sdk
```

Comment:
Github seems unhappy about this for some reason

Reply:
Yeah... That's because group 'temporal/server [doesn't have write access on this repo](https://github.com/orgs/temporalio/teams/server/repositories). I think we can safely ignore this.